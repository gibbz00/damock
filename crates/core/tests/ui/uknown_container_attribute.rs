#[derive(mock_default::Mock)]
#[mock(true)]
enum Foo {
    #[mock]
    Bar,
}

fn main() {}
