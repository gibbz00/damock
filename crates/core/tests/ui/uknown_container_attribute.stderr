error: expected identifier, found keyword `true`
 --> tests/ui/uknown_container_attribute.rs:2:8
  |
2 | #[mock(true)]
  |        ^^^^
